backend/lucene/src/main/java/org/hibernate/search/backend/lucene/logging/impl/Log.java

```diff
@@ -734,4 +734,8 @@ SearchException vectorKnnMatchVectorTypeDiffersFromField(String absoluteFieldPat
 
 	@Message(id = ID_OFFSET + 180, value = "An error occurred while parsing the query string '%1$s': %2$s")
 	SearchException queryStringParseException(String query, String message, @Cause Exception cause);
+
+	@Message(id = ID_OFFSET + 181,
+			value = "An unexpected attempt to create additional predicate filters (tenant/routing etc.).")
+	SearchException predicateRequestContextUnawareOfFilters();
```

Comment:
If this can result from a user errors, we need an actionable message: what did they do wrong, how to make it right?
If this cannot result from a user error, `AssertionFailure` would probably be a better fit (and wouldn't need to be declared in the logger).